apidoc/Titanium/WatchSession/WatchSession.yml

```diff
@@ -212,14 +266,52 @@ events:
         summary: If the complication is enabled in the apple watch.
         type: Boolean
         platforms: [iphone]
+      - name: isActivated
+        summary: | 
+            If the apple watch is currently activated. Only available on iOS 9.3
+            and later. See <Ti.WatchSession.isActivated> for more infos.
+        type: Boolean
+        platforms: [iphone]
+      - name: activationState
+        summary: | 
+            Returns the current activation state of the watch.  Only available on iOS 9.3
+            and later. See <Ti.WatchSession.activationState> for more infos.
+        type: Number
+        constants: Titanium.WatchSession.ACTIVATION_STATE_*
+        platforms: [iphone]
   - name: reachabilitychanged
     summary: |
         The watch reachability state has changed.
     properties:
+      - name: isPaired
+        summary: If the device is paired with the apple watch.
+        type: Boolean
+        platforms: [iphone]
       - name: isReachable
         summary: If apple watch is currently reachable.
         type: Boolean
         platforms: [iphone]
+      - name: isWatchAppInstalled
+        summary: If the watch app is installed in the apple watch.
+        type: Boolean
+        platforms: [iphone]
+      - name: isComplicationEnabled
+        summary: If the complication is enabled in the apple watch.
+        type: Boolean
+        platforms: [iphone]
+      - name: isActivated
+        summary: | 
+            If the apple watch is currently activated. Only available on iOS 9.3
+            and later. See <Ti.WatchSession.isActivated> for more infos.
+        type: Boolean
+        platforms: [iphone]
```

Comment:
That's what I thought. But the `osver` key is not allowed in event properties:
```yml
[ERROR] ./Titanium/WatchSession/WatchSession.yml
Titanium.WatchSession: found 1 error(s)!
	events
		watchstatechanged
			properties
				isActivated
					Invalid key: osver
```
So this is the way other event properties did it in the past. Let me know if we need to update the docs generation for this.